cgi-bin/DW/Widget/LatestInbox.pm

```diff
@@ -28,24 +28,9 @@ sub render_body {
 
     my $remote = LJ::get_remote()
         or return "";
+    my $vars = {};
 
-    my $limit = $opts{limit} || 5;
-
-    my $ret = "<h2>" . $class->ml('widget.latestinbox.title') . "</h2>";
-
-    $ret .= "<div class='sidebar'><ul>";
-    $ret .= "<li><a href='/inbox/'>" . $class->ml('widget.latestinbox.links.inbox') . "</a></li>";
-    $ret .=
-          "<li><a href='/inbox/compose'>"
-        . $class->ml('widget.latestinbox.links.compose')
-        . "</a></li>";
-    $ret .=
-          "<li><a href='/manage/settings/?cat=notifications'>"
-        . $class->ml('widget.latestinbox.links.manage')
-        . "</a></li>";
-    $ret .= "</ul></div>";
-
-    $ret .= "<div class='contents'>";
+    my $vars->{limit} = $opts{limit} || 5;
```

Comment:
These two lines can be combined as:
`my $vars = { limit => $opts{limit} || 5 };`

As written, the second `my` is a syntax error.